### ISSUE TITLE
Suggestion to create a tag for v1.0.0

### ISSUE THREAD

**Issue by arohm-ff** [O-X-L/ansible-collection-systemd#4]
Hi, I ran into an issue with initial setup for this module and am hoping to help future folks avoid that difficulty.

## Environment
collection_systemd v1.0.0
ansible [core 2.18.0]
python 3.13.0
Apple M1 chip running macOS 14.7
(please let me know if there are more useful versions missing)

## Expected Behavior
When I install the stable version of the project using the instructions from the installation guide (`ansible-galaxy collection install ansibleguy.systemd`), it should be able to run. If it's running into an error, I should be able to inspect the source in Github to look for the error.

## Observed Behavior
I installed using the command mentioned above, and encountered the following error:
```
{"changed": false, "module_stderr": "Shared connection to <IP address> closed.\r\n", "module_stdout": "/bin/sh: 1: /usr/bin/env python3: not found\r\n", "msg": "The module failed to execute correctly, you probably need to set the interpreter.\nSee stdout/stderr for the exact error", "rc": 127}
```

I tried suggestions I [found elsewhere](https://stackoverflow.com/questions/57167948/module-failed-to-start-ansible), of setting the `ansible_python_interpreter=/usr/bin/python3` or setting a specific interpreter for my target host, but that did not resolve the issue.

Ultimately I discovered that the version I had installed, 1.0.0, was not tagged in this repo, but that it was created prior to [this change to the module](https://github.com/ansibleguy/collection_systemd/commit/03dd13da2309dcf2a9396f64bf29b640fe2b9c79). Applying this change to the module in my local Ansible collection location enabled me to get it up and running. 

Also wanted to note that this project seems very useful - thank you for building it! I am also fairly new to Ansible so that may be a contributing factor, perhaps this is a common issue with migrating to newer modules and I just learned the hard way.

**Comment by superstes** [comment]
Hope I find time next week to address this issue.
Thank you for reporting it 👍

**Comment by ansibleguy** [comment]
The current codebase has been released as 1.0.1
https://galaxy.ansible.com/ui/repo/published/ansibleguy/systemd/

Thank your for informing me - have forgot to release it :/
